src/main/java/com/codeborne/selenide/Clipboard.java

```diff
@@ -0,0 +1,59 @@
+package com.codeborne.selenide;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nonnull;
+import java.awt.Toolkit;
+import java.awt.datatransfer.DataFlavor;
+import java.awt.datatransfer.StringSelection;
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Clipboard {
+
+  private Driver driver;
+
+  public Clipboard() {
+  }
+
+  public Clipboard(Driver driver) {
+    this.driver = driver;
+  }
+
+  /**
+   * Get text from clipboard
+   *
+   * @return
+   */
+  @Nonnull
+  @CheckReturnValue
+  public String getText() {
+    String content = null;
+    try {
+      content = Toolkit.getDefaultToolkit().getSystemClipboard().getData(DataFlavor.stringFlavor).toString();
+    } catch (UnsupportedFlavorException | IOException e) {
+      throw new IllegalStateException("Can't get clipboard data! " + e);
+    }
+    return content;
+  }
+
+  /**
+   * Set value to clipboard
+   *
+   * @param text value to be set to clipboard
+   */
+  public void setValue(String text) {
+    Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(text), new StringSelection(text));
+  }
+
+  /**
+   * Check that value in clipboard equals to expected
+   *
+   * @param text expected value for compare
+   */
+  public void shouldBeText(String text) {
+    assertEquals(text, getText(), "Clipboard data doesn't match with expected!");
```

Comment:
It should not be Clipboard method. it must be separate condition class extends `Condition.java`

Reply:
Ohh it would be nice. But it seems we don't have any Conditions that doesn't depend on element `$("")`

Reply:
I also want to use Condition but Clipboard is separate instance and it not depends from element)

Reply:
@dbudim I would suggest not add condition in this PR and create separate issue to discuss how we can implement such conditions. 
As I remembered we already had requests to have condition for e.g. current url like `smth.shouldHave(currentUrl("https://twitter.com/selenide"))`